ansible/slave.yml

```diff
@@ -7,10 +7,10 @@
    - jenkins_user: 'jenkins-build'
   tasks:
     - name: create a jenkins-build user
-      user: name=jenkins-build comment="Jenkins Build Slave User"
+      user: name=jenkins-build comment="Jenkins Build Slave User" home="/home/{{ jenkins_user }}"
```

Comment:
Should we do ``name="{{ jenkins_user }}"`` here instead of hardcoding jenkins-build?